project/Helpers.scala

```diff
@@ -173,17 +173,17 @@ object C {
       port: Int,
       componentName: String
   ): Seq[Def.Setting[_ >: Task[Seq[CmdLike]] with String with Boolean with Seq[Int] with Option[String]]] = Seq(
-    dockerBaseImage := "alpine:latest",
+    dockerBaseImage := "ghcr.io/graalvm/graalvm-ce:ol7-java17-22.3.3",
     dockerUsername := sys.props.get("docker.username"),
     dockerRepository := sys.props.get("docker.registry"),
     dockerUpdateLatest := true,
     dockerExposedPorts ++= Seq(port),
-    dockerCommands ++= Seq(
-      Cmd("FROM", componentName),
-      Cmd("USER", "root"),
-      ExecCmd("RUN", "/sbin/apk", "add", "--no-cache", "bash"),
-      ExecCmd("RUN", "/sbin/apk", "add", "--no-cache", "openjdk17-jre")
-    )
+    // dockerCommands ++= Seq(
```

Comment:
This looks like it's not needed any more with the new base image

Reply:
Can take it out with the next PR